backend/curation/curationlib/update_uei_or_ein.py

```diff
@@ -0,0 +1,144 @@
+from audit.models import SingleAuditChecklist, User, SubmissionEvent
+from audit.models.viewflow import SingleAuditChecklistFlow
+from curation.curationlib.curation_audit_tracking import CurationTracking
+
+import logging
+import sys
+from django.db.models import Q
+from django.db import transaction
+
+from dissemination.models import _dissemination_models
+
+
+logger = logging.getLogger(__name__)
+
+
+# FIXME: Should this be added to `naming.py`
+# so we can take a name and convert it to a model?
+def get_named_models():
+    # return {
+    #     "AdditionalEins": AdditionalEin,
+    #     "AdditionalUeis": AdditionalUei,
+    #     "CorrectiveActionPlan": CapText,
+    #     "FederalAwards": FederalAward,
+    #     "FindingsText": FindingText,
+    #     "FindingsUniformGuidance": Finding,
+    #     "NotesToSefa": Note,
+    #     "SecondaryAuditors": SecondaryAuditor,
+    #     "General": General,
+    #     "Passthrough": Passthrough,
+    # }
+    return _dissemination_models
+
+
+def get_named_parts_containing_ueis(sac):
+    return {
+        "AdditionalEins": sac.additional_eins,
+        "AdditionalUeis": sac.additional_ueis,
```

Comment:
```suggestion
        "AdditionalEINs": sac.additional_eins,
        "AdditionalUEIs": sac.additional_ueis,
```
Tested on full-submission submissions